docs/getting-started/summaryofchanges.md

```diff
@@ -44,6 +44,14 @@ The following features and enhancements were added.
 
 #### API Mediation Layer
 
+- Provided Zowe Authentication and Authorization Service (ZAAS) client
+- Refreshed the static client definitions from the API Catalog UI
+- Switched to sso-auth instead of apiml-auth
+- Added logout endpoint api doc
+- Made jjwt only a test dependency
+- Fixed order of fetching JWT from request
+- Implemented request retrying for service instances
```

Comment:
- Is there a related GitHub issue or Pull Request for each entry? 
  > Made jjwt only a test dependency  

Just want to confirm that jjwt is not a typo. : )

Reply:
I found that it is not a typo, it is the real name of a dependency Java JSON Web Tokens 
https://github.com/jwtk/jjwt